README.md

```diff
@@ -246,33 +268,24 @@ controller.on('message_received', processWatsonResponse);
 Events are messages having type different than `message`.
 
 [Example](https://github.com/howdyai/botkit/blob/master/examples/facebook_bot.js) of handler:
+
```

Comment:
Example link above is outdated and broken, here is the right one: https://github.com/howdyai/botkit/blob/master/packages/docs/reference/facebook.md#facebookeventtypemiddleware